test/mock_data/job.py

```diff
@@ -0,0 +1,35 @@
+class job:
```

Comment:
I don't see this used anywhere.

Reply:
This is used in the cassette setup test that is commented out.